ehsm_kms_service/secret_manager_apis.js

```diff
@@ -265,9 +265,69 @@ const createSecret = async (res, appid, payload, DB) => {
     }
 }
 
+//update the description of secret
+const updateSecretDesc = async (res, appid, payload, DB) => {
+    //get and check param in payload
+    let secretName = getParam_String(payload, 'secretName')
+    let description = getParam_String(payload, 'description')
+    if (!checkStringParam(secretName, true, SECRETNAME_LENGTH_MAX)) {
+        res.send(_result(400, 'secretName cannot be empty, must be string and length not more than 64'))
+        return
+    }
+    if (!checkStringParam(description, true, DESCRIPTION_LENGTH_MAX)) {
```

Comment:
empty description '' still needs support.

Reply:
ok, it will be add